File: src/main/java/ru/mdashlw/hypixel/reply/impl/GuildReply.java
package ru.mdashlw.hypixel.reply.impl;

import com.fasterxml.jackson.annotation.JsonProperty;
import ru.mdashlw.hypixel.entities.Guild;
import ru.mdashlw.hypixel.reply.Reply;

public class GuildReply extends Reply<Guild> {
    @JsonProperty
    private Guild guild;

    @Override
    public Guild getValue() {
        return guild;
    }
}

File: src/main/kotlin/ru/mdashlw/hypixel/reply/impl/GuildReply.kt
package ru.mdashlw.hypixel.reply.impl

import ru.mdashlw.hypixel.elements.Guild
import ru.mdashlw.hypixel.reply.Reply

class GuildReply(
    success: Boolean,
    cause: String?,
    throttle: Boolean,
    guild: Guild?
) : Reply<Guild>(success, cause, throttle, guild)
